tutorials/video_recorder.md

```diff
@@ -0,0 +1,107 @@
+\page videorecorder Video Recorder
+
+## Using the video recorder plugin
+
+Ignition Gazebo offers a video recorder tool for recording videos from the 3D
+Scene. The tool is available as a GUI plugin. To open this plugin, first launch
+Ignition Gazebo and select the ellipsis menu on top right (3 dots menu), and
+scroll down to find the `Video Recorder` option. Click on the plugin to open
+the Video Recorder tool. Alternatively, launch the demo world in Ignition Gazebo
+that already has this plugin included in the GUI.
+
+```
+ign gazebo -v 4 video_record_dbl_pendulum.sdf
+```
+
+In this plugin, you should see a single button with a video recorder icon.
+Clicking on the button gives you the video format options that are available.
+
+@image html files/video_recorder/video_recorder.png
+
+Once an option is selected, recording starts immediately as indicated by
+a flashing video recorder icon. At anytime that you wish to stop recording,
+click on the flashing icon and select `Stop`. A file dialog window should pop up
+and let you select the path to save the recorded video in. Save the file and
+make sure to also include the file extension in the filename.
```

Comment:
Manually specifying the file extension is not required, right? I think John fixed this.

Reply:
yep just verified that it works without specifying the extension. bdee780300291a93a6ccc239e65640ca3201276f